### PR TITLE
Unified Theming

### PR DESCRIPTION
**Pull request by pygaurav**
In this PR, the theme has been made fully unified. Since the essence of the theme is based on dark colour preferences, hence the unified theming approach has been followed.

### CODE REVIEW

**Comment by robmonte** [comment]
This looks great! I have 2 comments that I've noticed so far, let me know if you agree or disagree.

1. I personally prefer the separator between the editor and both the left/right sidebars should be a lighter color and more visible, as it's hard to see at least to my eyes. If this isn't your desire for the theme that's understandable.
2. The selected editor tab could be lighter as well, to make it more visually distinct from the background tabs.

**Comment by pygaurav** [comment]
@robmonte I agree with your points 👍, but I am selecting a lighter color that must be complementary to the overall dark theme. The lighter color is still not finalized yet 😶‍🌫️. As this PR is not yet completed, I have taken your feedback to a note.

**Comment by pygaurav** [comment]
@robmonte I agree with your points 👍, but I am still in process of selecting a lighter color (will not be that light) that must be complementary to the overall dark theme. As this PR is not yet completed, I have taken your feedback to a note. I am strongly in favor of keeping the tab color similar to the theme's primary color, instead of color we can have a lighter border that will be a differentiation factor.

File: themes/vasuki-dark-theme.json [hunk 1]
```diff
@@ -1,589 +1,560 @@
...
+    "titleBar.inactiveForeground": "#6b717d",
+    "titleBar.activeBackground": "#0C0E14",
+    "titleBar.inactiveBackground": "#0C0E14",
+    "notification.background": "#0C0E14",
+    "statusBar.debuggingBackground": "#020304",
+    "menu.selectionBackground": "#141721",
+    "menu.selectionForeground": "#d7dae0",
+    "checkbox.background": "#383e4a",
+    "panelSectionHeader.background": "#0C0E14",
+    "panelSectionHeader.border": "#fff",
```
Comment (robmonte) [inline themes/vasuki-dark-theme.json:68]: Should `"panelSectionHeader.border"` be set to this color? Seems like it should be `#293045` to match the rest of the theme.

Comment (pygaurav) [reply themes/vasuki-dark-theme.json:68]: Ahh, Thanks! Nice catch :see_no_evil: I have changed the same on the parent branch. Release `v1.0.6` is live now :rocket: .

Comment (robmonte) [reply themes/vasuki-dark-theme.json:68]: Haha no problem, sorry for so many changes! Thanks for the theme! The updates you made look really good.

Comment (pygaurav) [reply themes/vasuki-dark-theme.json:68]: I appreciate your changes and thank you so much for taking time for making the theme better :+1:

**Pull request merged**